src/actions/GroupActions.js

```diff
@@ -18,6 +18,14 @@ import { asyncAction } from './actionCreators';
 
 const GroupActions = {};
 
+/**
+ * Create a GroupActions.fetchJoinedGroups action that represents an
```

Comment:
not aiui.

It returns an action creator: ie, a function which, when called, will start such a request and create such a (set of) actions.